.github/workflows/continuous-integration.yml

```
@@ -15,10 +15,10 @@ jobs:
       - name: Execute linters and test suites
         run: ./docker/cibuild
```

As `continuous-integration.yml` is launching `coverage` from within a docker container (`docker/cibuild`), the codecov-action silently fails, without finding the coverage file.

---

issue is deferred via #190